src/uu/stat/src/stat.rs

```diff
@@ -93,9 +97,30 @@ pub enum OutputType {
     Unknown,
 }
 
+enum QuotingStyle {
+    Locale,
+    Shell,
+    Default,
```

Comment:
I would rename this variant to `ShellEscapeAlways` because `shell-escape-always` is the default. And it might make sense to add `#[derive(Default)]` and mark this variant as `#[default]`.